### PR TITLE
NGDOC-75 Document for cache time

### PR DESCRIPTION
**Pull request by nibinqtl**

### CODE REVIEW

File: docs/edge-logic/faq.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
 ## Frequently Asked Questions
 
+### How the cache time for an object is determined?
+
+There are many directives you can use in the Edge Logic to control the cache time. If none of them is configured, the default behavior of CDN360 edge servers is "honor the origin". That is, the instructions in the `Cache-Control` and `Expires` header fields are followed. If these two headers are not present in a response from the origin, the response is not cached. In the meantime, the presence of the `Set-Cookie` header also prevents caching of the response. 
+
+We modified the open-source NGINX to strictly follow the HTTP standard regarding 'zero-time cache'. When `no-cache` or `max-age=0` is present in the `Cache-Control` header field, the response is still cached, but expires immediately. Any subsequent request for this object will result in a revalidation request to the origin with the `If-Modified-SInce` header. If `no-store` is in the `Cache-Control` header field, the response will not be cached.
```
Comment (burtonquantil) [inline docs/edge-logic/faq.md:7]: Omit the comma before "but expires immediately".

File: docs/edge-logic/faq.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+
+By default, the `Date` header field is passed all the way from the origin to the edge server, and efforts have been made to ensure the `Age` header field reflects the time since the response is retrieved from the origin, even when parent cache is used.
+
+If the above mentioned default behavior is not meeting your requirement, you can use the following directives to alter it.
```
Comment (burtonquantil) [inline docs/edge-logic/faq.md:11]: Suggestion:   If the default behavior does not meet your requirements, use the following directives to alter it.

**Review by burtonquantil — CHANGES_REQUESTED** [review]
Only a couple of minor suggestions.

File: docs/edge-logic/faq.md (hunk 1 above)
Comment (marcfishelquantil) [reply docs/edge-logic/faq.md:7]: In the second sentence, please add the word "to" between "is" and "honor." So the sentence reads "...the default behavior of CDN360 edge servers is to "honor the origin"."
In the last sentence in the first paragraph, please delete "In the meantime," as it adds nothing to the sentence.

File: docs/edge-logic/faq.md (hunk 2 above)
Comment (marcfishelquantil) [reply docs/edge-logic/faq.md:11]: The first sentence is pretty long. How about breaking it into 2 sentences: "...to the edge server. Efforts have  been made..."
In the second sentence, please add a comma between "above" and "mentioned." How about rewording this sentence as follows: "If the default behavior mentioned above does not meet your requirement, use the following directives to change the behavior."

File: docs/edge-logic/faq.md (hunk 1 above)
Comment (nibinqtl) [reply docs/edge-logic/faq.md]: Thanks, updated.

File: docs/edge-logic/faq.md (hunk 2 above)
Comment (nibinqtl) [reply docs/edge-logic/faq.md]: Thanks, updated.

**Comment by nibinqtl** [comment]
@burtonquantil @marcfishelquantil , please take another look.

**Review by burtonquantil — APPROVED** [review]
OK to merge.
